etl/steps/data/explorers/un/2022/un_wpp.py

```diff
@@ -29,7 +29,33 @@ def _keep_relevant_rows(df: pd.DataFrame) -> pd.DataFrame:
     df = df.reset_index()
     df = df.loc[
         df.variant.isin(["estimates", "low", "medium", "high"])
-        & -(df.metric.isin(["net_migration", "net_migration_rate"]) & (df.location == "World"))
+        & -(
```

Comment:
very minor, I generally prefer using `~` than `-` as the negation operator. I think is better practice, but I just can't remember why : D

Reply:
Ah okay - I will update it to that!